api/v2/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/decred/dcrtime/api/v2
```

Comment:
Why is there a go.mod file in this package?

Reply:
Marco expressed that he'd want third-party consumers to import only the api v2 package, without being tied to the daemon. For that we should mod the api directory. From what I understood about this, I think we also would have to push a tag in the repo, `api/v2.0.0`, so that it is available on github to be properly downloaded by third-parties. Once this is done, we could remove the `replace github.com/decred/dcrtime/api/v2 => ./api/v2` that is on dcrtime's `go.mod`, bc go would directly link the `require github.com/decred/dcrtime/api/v2 v2.0.0` to github. Similar to what dcrd is doing to mod its packages, if I'm not mistaken.